webapp/src/libs/auth/AuthHelper.ts

```diff
@@ -9,36 +9,27 @@ import {
 } from '@azure/msal-browser';
 import debug from 'debug';
 import { Constants } from '../../Constants';
-import { ActiveUserInfo } from '../../redux/features/app/AppState';
-import { IChatUser } from '../models/ChatUser';
+import { store } from '../../redux/app/store';
 import { TokenHelper } from './TokenHelper';
 
 const log = debug(Constants.debug.root).extend('authHelper');
 
 export const enum AuthType {
+    None = 'None',
     AAD = 'AzureAd',
 }
 
-// This is the default user information when authentication is set to 'None'.
-// It must match what is defined in PassthroughAuthenticationHandler.cs on the backend.
-export const DefaultChatUser: IChatUser = {
-    id: 'c05c61eb-65e4-4223-915a-fe72b0c9ece1',
-    emailAddress: 'user@contoso.com',
-    fullName: 'Default User',
-    online: true,
-    isTyping: false,
-};
-
-export const DefaultActiveUserInfo: ActiveUserInfo = {
```

Comment:
Did you mean to remove this? Isn't it still referenced as the default in AuthHelper?